package.json

```diff
@@ -1,23 +1,30 @@
 {
   "name": "@iden3/js-crypto",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
I guess you could leave the version as 0.0.1 cause this package is not published yet